lib/flutter_local_notifications.dart

```diff
@@ -10,6 +10,20 @@ typedef Future<dynamic> MessageHandler(String message);
 /// The available intervals for periodically showing notifications
 enum RepeatInterval { EveryMinute, Hourly, Daily, Weekly }
 
+class Time {
+  final int hour;
+  final int minute;
+  final int second;
+
+  Time([this.hour = 0, this.minute = 0, this.second = 0]);
+
+  Map<String, int> get serialized => {
```

Comment:
if you could make this a toMap() method to be consistent with the rest of the code then that would be great as reading something like `time.serialized` feels awkward